src/lsp/LSPClient.js

```diff
@@ -299,7 +299,7 @@ export default class LSPClient {
       },
       position: {
         line: iter_cursor.get_line(),
-        character: iter_cursor.get_line_offset() - 1,
+        character: iter_cursor.get_line_offset(),
```

Comment:
> `character: integer` Character offset on a line in a document (zero-based)

from the spec: https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#position

Since `iter_cursor.get_line_offset()` is also zero-based, removing one caused it to be -1